msteams-platform/get-started/prerequisites.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Get started - Prerequisites
+author: adrianhall
+description: Learn how to get started with Microsoft Teams app development and set up your environment.
+ms.author: adhal
+ms.date: 05/18/2021
+ms.topic: quickstart
+---
+# Prerequisites: Get started with Microsoft Teams app development
+
+Before your create your first Teams app, you must install a few tools and set up your development environment.
+
+## Install required tools
+
+Some of the tools you need depend on how you you prefer to build your Teams app:
+
+- [Node.js](https://nodejs.org/en/download/) (use the latest v14 LTS release) 
+- A browser with developer tools - such as [Microsoft Edge](https://www.microsoft.com/edge) (recommended) or [Google Chrome](https://www.google.com/chrome/)
+- If you're developing with JavaScript, TypeScript, or the SharePoint Framework (SPFx), install [Visual Studio Code](https://code.visualstudio.com/download), version 1.55 or later.  
+- If you' a're developing with .NET, install [Visual Studio 2019](https://visualstudio.com/download).  Ensure you install the **ASP.NET and web development** or **.NET Core cross-platform development** workload.
+
+> [!WARNING]
+> There are known issues with `npm@7`, packaged with Node v15 and later. If you have problems running `npm install`, ensure you're using Node v14 (LTS)
+
+## Install the Teams Toolkit
+
+The Teams Toolkit helps simplify the development process with tools to provision and deploy cloud resources for your app, publish to the Teams store, and more. You can use the toolkit with Visual Studio Code, Visual Studio, or as a CLI (called `teamsfx`).
+
+# [Visual Studio Code](#tab/vscode)
+
+1. Open Visual Studio Code.
+1. Select the Extensions view (**Ctrl+Shift+X** / **⌘⇧-X** or **View > Extensions**).
+1. In the search box, enter _Teams Toolkit_.
+1. Select on the green install button next to the Teams Toolkit.
+
+You also can find the Teams Toolkit on the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension).
+
+The following tools will be installed by the Visual Studio Code extension when they are needed.  If already installed, the installed version will be used instead.  If using Linux, you must install these tools before use:
```

Comment:
> If already installed, the installed version will be used instead.

It's not accurate since **Azure Functions Core Tools** will be installed as dev dependency even it's installed globally on user's machine, and the global installed `Func Tool` won't be used.

Reply:
> The following tools will be installed by the Visual Studio Code extension when they are needed.

But this is **NOT TRUE** for Linux, users still need to install these dependencies by themselves.